src/index.js

```diff
@@ -449,11 +449,15 @@ function createConfig(options, entry, format, writeMeta) {
 							sourcemap: true,
 							output: { comments: false },
 							compress: {
+								passes: 10,
+								hoist_funs: true,
+								booleans_as_integers: true,
```

Comment:
Based on its description this seems like really unsafe

Reply:
removed it and added an escape hatch so folks can specify `compress:{}` in the existing package.json mangle configuration.